api/lib/searchOperators.js

```diff
@@ -69,6 +92,38 @@ const getQueryConditionsAndParams = (
   };
 };
 
+const hasBindiCharacter = charCode => {
+  if (bindiCharacters[charCode]) {
+    return bindiCharacters[charCode];
+  }
+  return false;
+};
```

Comment:
Can we do 

`const hasBindiCharacter = charCode => bindiCharacters[charCode] || false;`

Reply:
Fixed.